### PR TITLE
Avocado Website Update

### PR DESCRIPTION
**Pull request by harvey0100**
Avocado website updated according to designs provided lots of new content such as updated utils table.
Many changes to the Main Avocado Framework Page included. Other repos such as Aexpect / VT links point to their repo.

### CODE REVIEW

File: CREDITS
Comment (richtja) [inline CREDITS:1]: IMO you should keep the credits, because those new designs and icons has been created by community-design-team, and we should add their credits. I think adding [this](https://gitlab.com/fedora/design/community-design-team/issues/-/issues/146/) and listing the creators will be enough.

File: LICENSE
Comment (richtja) [inline LICENSE:1]: We need to keep the LICENSE.

File: _config.yml
Comment (richtja) [inline _config.yml:30]: IMO, these configs are leftovers from your testing, am I right?

File: _data/versions.yml
Comment (richtja) [inline _data/versions.yml:1]: Because we just made release 108. This needs an update.

File: _data/modules.yml
Comment (richtja) [inline _data/modules.yml:1]: Shouldn't this file be generated during the deployment from autils repo?

File: resources/images/cleber.jpg
Comment (richtja) [inline resources/images/cleber.jpg:1]: IMO, these profile pictures, shouldn't be part of this repo. We should fetch them from github during the deployment, because the number of maintainers will increase when the autils will start and we need to have this as automated as possible. If this will be too difficult, we can have the autils page without profile pictures.

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @harvey0100, thank you very much for this, the new page design is awesome. I wasn't able to properly review the overall implementation because I don't have enough experience with web development. But I have few practical comments related to future maintainability of this page. Also, can you please test this with current [build process](https://github.com/avocado-framework/avocado-framework.github.io/blob/master/.github/workflows/jekyll-gh-pages.yml) to make sure that this will be properly deployed after merge. Thanks

File: CREDITS
Comment (harvey0100) [reply CREDITS:1]: Resolved

File: _data/modules.yml
Comment (harvey0100) [reply _data/modules.yml]: New workflow created that takes this into account and applies avatars.

File: resources/images/cleber.jpg
Comment (harvey0100) [reply resources/images/cleber.jpg:1]: New workflow created that takes this into account and applies avatars.

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @harvey0100, thank you for the updates. I saw that you add the requested changes to the new commit, please squash this commit to the eeaffa04b45d00c751bb1169dfbd0086c52604af.  Overall, the updates LGTM. Can you also run the deployment workflow on your fork and share the results? I would like to make sure that it is properly tested, before we will merge it.

**Comment by harvey0100** [comment]
Hi @richtja 

I've squashed the commits as requested the workflow is running correctly on deployment and build you can view the results from my fork here https://github.com/harvey0100/harvey0100.github.io/actions/runs/11440045052/job/31825025214 - The only further change we will have to do is add a maintainer image field in ar.yml in autils as per my file here in my fork https://github.com/harvey0100/autils/blob/modules/metadata/autils/archive/ar.yml

**Comment by harvey0100** [comment]
Added a YML Merge step in the workflow which will merge all YML moduels from Autils into a singular file during the build process which Jekyll will reference for the Autils module table.

**Comment by richtja** [comment]
> Hi @richtja
> 
> I've squashed the commits as requested the workflow is running correctly on deployment and build you can view the results from my fork here https://github.com/harvey0100/harvey0100.github.io/actions/runs/11440045052/job/31825025214 - The only further change we will have to do is add a maintainer image field in ar.yml in autils as per my file here in my fork https://github.com/harvey0100/autils/blob/modules/metadata/autils/archive/ar.yml

Hi @harvey0100, thank you for the update and the testing, we are getting there. 

To be honest, I don't understand why we need to update the ar.yml in autils for generating images. Wouldn't be possible to generate the maintainer image field in `deployavatar.py` script? IMO, it is slightly strange to have empty field in yaml just for processing purposes.

File: scripts/deployavatar.py
Comment (richtja) [inline scripts/deployavatar.py:51]: Can you please explain to me, why we need the `or` statement. IMO it should work only with `github_usr_name` if not there is some issue in yaml parser.

**Comment by harvey0100** [comment]
Closing due to merging to master moving over to new PR - https://github.com/avocado-framework/avocado-framework.github.io/pull/29

**Pull request closed without merging**
